cli/lib/tasks/install.js

```diff
@@ -336,104 +247,97 @@ const start = (options = {}) => {
       return true
     }
 
-    if ((binaryVersion === needVersion) || !util.isSemver(needVersion)) {
+    if ((binaryVersion === versionToInstall) || !util.isSemver(versionToInstall)) {
       // our version matches, tell the user this is a noop
       alreadyInstalledMsg()
 
       return false
     }
 
     return true
-  })
-  .then((shouldInstall) => {
-    // noop if we've been told not to download
-    if (!shouldInstall) {
-      debug('Not downloading or installing binary')
+  }
 
-      return
-    }
+  // noop if we've been told not to download
+  if (!shouldInstall()) {
+    return debug('Not downloading or installing binary')
+  }
 
-    if (needVersion !== pkgVersion) {
-      logger.log(
-        chalk.yellow(stripIndent`
-          ${logSymbols.warning} Warning: Forcing a binary version different than the default.
+  if (versionToInstall !== pkgVersion) {
```

Comment:
Will this alway log for binary version installs?

Reply:
Yup. The log message here isn't great, I was kinda just following what was here already, but I can improve it so that it indicates we're installing a pre-release and there's not a "different binary version"

Reply:
Log message updated, see snapshots. Also updated output for `cypress info`.